### PR TITLE
Update install and uninstall guides

### PR DESCRIPTION
**Pull request by davedoering**
Edited for clarity, active voice, and grammar.

### CODE REVIEW

File: content/docs/install/uninstallation_guide.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ weight: 3
 
 # Uninstallation Guide
 
-This guide describes how to uninstall Open Service Mesh (OSM) from a Kubernetes cluster. This guide assumes there is a single OSM control plane (mesh) running. If there are multiple meshes in a cluster, repeat the process described for each control plane in the cluster before uninstalling any cluster wide resources at the end of the guide. Taking into consideration both the control plane and dataplane, this guide aims to walk through uninstalling all remnants of OSM with minimal downtime.
+This guide describes how to uninstall Open Service Mesh (OSM) from a Kubernetes cluster. IT assumes you have a single OSM control plane (mesh) running. If you have multiple meshes in a cluster, repeat the process described below for each control plane in the cluster. Do this before uninstalling any cluster-wide resources described at the end of this guide. The steps here cover both the control plane and the dataplane. The goal is to take you through the uninstall of all OSM elements with minimal downtime.
```
Comment (bridgetkromhout) [inline content/docs/install/uninstallation_guide.md:11]: What is `IT`? Should this be "It"?

File: content/docs/install/uninstallation_guide.md [hunk 2]
```diff
@@ -59,51 +59,54 @@ Restart all pods running with a sidecar:
...
+
+in each namespace you removed from the mesh earlier.
 
-If there are ingress resources and they are configured to allow HTTPS traffic, SSL related annotations will need to be updated appropriately.
+If you have ingress resources and have them configured to allow HTTPS traffic, you will need to update the SSL-related annotations appropriately.
 
-Applications may be unavailable from outside the cluster for some time if ingress resources need to be reconfigured.
+> Note: Your applications may be unavailable from outside the cluster for some time if you need to reconfigure ingress resources.
 
 ### Delete Envoy Bootsrap Secrets
```
Comment (bridgetkromhout) [inline content/docs/install/uninstallation_guide.md:84]: `Bootsrap` should be "Bootstrap".

File: content/docs/install/manual_demo.md [hunk 3]
```diff
@@ -41,24 +41,24 @@ unzip osm.zip
...
 
-The `osm` CLI can be compiled from source using [this guide](/docs/install/).
+You can compile the `osm` CLI from source using [this guide](/docs/install/).
 
 
 
 ## Installing OSM on Kubernetes
 
-With the `osm` binary downloaded and unzipped, we are ready to install Open Service Mesh on a Kubernetes cluster:
+After you download and unsip the `osm` binary, you are ready to install Open Service Mesh on a Kubernetes cluster:
```
Comment (bridgetkromhout) [inline content/docs/install/manual_demo.md:50]: `unsip` should be "unzip"

File: content/docs/install/manual_demo.md [hunk 4]
```diff
@@ -101,13 +101,15 @@ state of allowed and blocked traffic between pods:
...
+To show the SMI Traffic Split, you need to deploy an additional application:
 
-- `bookstore-v2` - this is the same container as the first `bookstore` we deployed, but for this demo we will assume that it is a new version of the app we need to upgrade to.
+- `bookstore-v2` - This is the same container as the first `bookstore` you deployed, but for this demo you can assume that it is a new version of the app you need to upgrade to.
 
 The `bookbuyer`, `bookthief`, `bookstore`, and `bookwarehouse` Pods will be in separate Kubernetes Namespaces with
 the same names. Each new Pod in the service mesh will be injected with an Envoy sidecar container.
 
+> Note: The following commands *must* be run in Bash and *not* in Powershell.
+
```
Comment (bridgetkromhout) [inline content/docs/install/manual_demo.md:112]: As the commands will work in other unix shells (like zsh), I would rephrase this to say a Linux shell or compatible (as with WSL2).

**Comment by bridgetkromhout** [comment]
A few things to fix before we can merge:

- resolve the merge conflicts.
- add the signoff line to your updated commits.
- correct the items I noted.

Thanks!

File: content/docs/install/manual_demo.md [hunk 5]
```diff
@@ -41,24 +41,24 @@ unzip osm.zip
...
+The Install command enables:
 [Prometheus](https://github.com/prometheus/prometheus),
 [Grafana](https://github.com/grafana/grafana), and
 [Jaeger](https://github.com/jaegertracing/jaeger) integrations.
 The `OpenServiceMesh.enablePermissiveTrafficPolicy` chart value instructs OSM to ignore any policies and
 let traffic flow freely between the pods. With Permissive Traffic Policy mode enabled, new pods
 will be injected with Envoy, but traffic will flow through the proxy and will not be blocked.
 
-> Note: Permissive Traffic Policy mode is an important feature for brownfield deployments, where it may take some time to craft SMI policies. While operators design the SMI policies, existing services will to continue to operate as they have been before OSM was installed.
+> Note: Permissive Traffic Policy mode is an important feature for brownfield deployments (where it may take some time to craft SMI policies). Operators can design the SMI policies while existing services continue to operate as they have been before you installed OSM.
```
Comment (bridgetkromhout) [inline content/docs/install/manual_demo.md:61]: On line 65, we have
```
    --set=OpenServiceMesh.enablePermissiveTrafficPolicy=true
```

Does that work as expected for you? AFAICT there is a missing trailing `\`.

File: content/docs/install/uninstallation_guide.md [hunk 6]
```diff
@@ -59,51 +59,54 @@ Restart all pods running with a sidecar:
...
-removed from the mesh earlier.
+You may have ingress resources in the cluster configured to allow traffic from outside the cluster to an application that was
+once part of the mesh. Identify these resources using the command:
+
+`kubectl get ingress -n <namespace>` 
+
+in each namespace you removed from the mesh earlier.
 
-If there are ingress resources and they are configured to allow HTTPS traffic, SSL related annotations will need to be updated appropriately.
+If you have ingress resources and have them configured to allow HTTPS traffic, you will need to update the SSL-related annotations appropriately.
```
[the remark this answers is not in the document]
Comment (davedoering) [reply content/docs/install/uninstallation_guide.md:80]: Yes, Phill. I think we could use diagrams to clarify these options..

File: content/docs/install/manual_demo.md (hunk 5 above)
Comment (davedoering) [reply content/docs/install/manual_demo.md:62]: I do see how subsequent commands do have a trailing \ but I didn't see any errors on that part of the install.

File: content/docs/install/manual_demo.md (hunk 4 above)
Comment (davedoering) [reply content/docs/install/manual_demo.md:113]: Done.

File: content/docs/install/manual_demo.md [hunk 7]
```diff
@@ -419,66 +425,66 @@ kubectl get meshconfig osm-mesh-config -n osm-system -o jsonpath='{.spec.traffic
...
 
-This can be demonstrated further by disabling permissive traffic policy mode and verifying that the counter for books bought from `bookstore` is not incrementing anymore:
+You can demonstrate this operation further by disabling permissive traffic policy mode. To verify the mode is disabled, check that the counter for books bought from `bookstore` no longer increments:
 
 ```bash
 kubectl patch meshconfig osm-mesh-config -n osm-system -p '{"spec":{"traffic":{"enablePermissiveTrafficPolicyMode":false}}}'  --type=merge
 ```
 
-_Note: When you disable permissive traffic policy mode, SMI traffic access mode is implicitly enabled. If counters for the books are incrementing then it could be because some SMI Traffic Access policies have been applied previously to allow such traffic._
+_Note: When you disable permissive traffic policy mode, SMI traffic access mode is implicitly enabled. If counters for the books increment, this could be because some SMI Traffic Access policies have been applied previously to allow such traffic._
```
[the remark this answers is not in the document]
Comment (davedoering) [reply content/docs/install/manual_demo.md]: Done.

File: content/docs/install/uninstallation_guide.md [hunk 8]
```diff
@@ -48,7 +48,7 @@ $ osm namespace remove <namespace> --mesh-name=<mesh-name>
 Namespace [<namespace>] successfully removed from mesh [<mesh-name>]
 ```
 
-Alternatively, if sidecar injection is enabled via annotations on pods instead of per namespace, please modify the pod or deployment spec to remove the sidecar injection annotation.
+Alternatively, if you enabled sidecar injection via annotations on pods instead of per namespace, please modify the pod or deployment spec to remove the sidecar injection annotation.
```
[the remark this answers is not in the document]
Comment (shashankram) [reply content/docs/install/uninstallation_guide.md:51]: I can help.

**Comment by bridgetkromhout** [comment]
Closing due to merge conflicts and lack of signoff.
